src/libraries/System.Formats.Nrbf/ref/System.Formats.Nrbf.Experimental.cs

```diff
@@ -0,0 +1,4 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+[assembly: System.Diagnostics.CodeAnalysis.ExperimentalAttribute("SYSLIB5005", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Huh, we don't have any kind of description on experimental that provides more detail in the diagnostic. I thought we did. Should we look at adding that? Someone can go to the docs, but it'd be nice if the message here could say that it's fully supported but the shape might change.

Reply:
I thought we did too until I did one of the earlier experimentals. But yeah, that API doesn't have a message attribute; it always uses a message from Roslyn. I'll submit an API suggestion to add it.